daemons/ipa-slapi-plugins/ipa-cldap/ipa_cldap_netlogon.c

```diff
@@ -350,6 +354,7 @@ int ipa_cldap_netlogon(struct ipa_cldap_ctx *ctx,
             ret = EINVAL;
             goto done;
         }
+        free(domain);
```

Comment:
This will lead to a double-free error. ``domain`` is already freed in the done block.